.yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs

```diff
@@ -0,0 +1,363 @@
+/* eslint-disable */
```

Comment:
Shouldn't these be ignored based on https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored ?

Reply:
This is the recommended .gitignore by yarn
````yml
.pnp.*
.yarn/*
!.yarn/cache
!.yarn/patches
!.yarn/plugins
!.yarn/releases
!.yarn/sdks
!.yarn/versions
````
It means all files under `.yarn` should be ignored, except the ones with `!`

I did the same, excpect for `.pnp` and `.yarn/cache`. because we don't use `pnp` and `.yarn/cache` makes so many commits and would fill PRs with useless data
https://github.com/curaOS/source/blob/902883c9f5f350b41a2d23d00aa6b4b2f0d4ef44/.gitignore#L14-L19

Reply:
aha, sorry; my bad!